protostar/configuration_file/configuration_file_v2_test.py

```diff
@@ -144,3 +138,31 @@ def test_saving_configuration(
 
     result = filepath.read_text()
     assert result == protostar_toml_content
+
+
+def test_saving_in_particular_order(
```

Comment:
I think 1. building the configuration file ad hoc specifically for this test and 2. testing exact contents would be more robust here and 3. testing only ConfigurationTOMLWriter independent of the model.

Reply:
> 3. testing only ConfigurationTOMLWriter independent of the model.

I'll remove `ConfigurationTOMLWriter`. `ConfigurationTOMLContentBuilder` is what should be tested.